src/main/java/seedu/duke/CommandParser.java

```diff
@@ -0,0 +1,142 @@
+package seedu.duke;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import seedu.duke.command.ListModules;
+import seedu.duke.command.DetailModules;
+import seedu.duke.command.AddUserModule;
+import seedu.duke.command.DeleteUserModule;
+import seedu.duke.command.Help;
+import seedu.duke.command.ViewGradRequirements;
+import seedu.duke.command.RecommendedSchedule;
+import seedu.duke.command.Specialisation;
+import seedu.duke.command.Workload;
+import seedu.duke.command.SuUserModule;
+
+
+import static seedu.duke.Duke.currentUser;
+
+public class CommandParser {
+
+    private static final Logger logger = Logger.getLogger(CommandParser.class.getName());
```

Comment:
This method is very long. Generally try to keep the method length to about 30 Lines of Code